aries-site/src/layouts/main/Layout.js

```diff
@@ -53,15 +61,79 @@ export const Layout = ({
   const layout = isLanding ? 'plain' : pageLayout;
 
   const MainContentWrapper = isLanding ? Fragment : PageContent;
-  const size = useContext(ResponsiveContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  const [value, setValue] = useState();
+  const [surveyResults, setSurveyResults] = useState({});
```

Comment:
Could be called `surveyFields`. surveyResults makes me think the actual survey answers you're posting back rather than just the field names.

Reply:
no longer used